src/neo/SmartContract/Native/Tokens/NeoToken.cs

```diff
@@ -133,15 +135,17 @@ public BigInteger UnclaimedGas(StoreView snapshot, UInt160 account, uint end)
         }
 
         [ContractMethod(0_05000000, ContractParameterType.Boolean, ParameterTypes = new[] { ContractParameterType.PublicKey }, ParameterNames = new[] { "pubkey" })]
-        private StackItem RegisterValidator(ApplicationEngine engine, Array args)
+        private StackItem RegisterCandidate(ApplicationEngine engine, Array args)
```

Comment:
We don't have `UnRegister Candidate` maybe some members want to leave the committee.

Reply:
Done.